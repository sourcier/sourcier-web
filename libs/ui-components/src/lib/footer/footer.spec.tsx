import renderer from 'react-test-renderer';

import Footer from './footer';

describe('Footer', () => {
  it('should render successfully', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
