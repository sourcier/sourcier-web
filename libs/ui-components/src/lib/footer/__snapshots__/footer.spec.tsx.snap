// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render successfully 1`] = `
<div
  className="bg-base-200 text-base-content"
>
  <div
    className="max-w-6xl px-4 mx-auto my-16"
  >
    <div
      className="flex flex-col text-center md:flex-row md:text-left"
    >
      <div
        className="mb-8 md:mb-0"
      >
        <p>
          Made with
           
          <span
            aria-label="Heart emoji"
            role="img"
          >
            ❤️
          </span>
           
          in London, UK.
        </p>
        <p>
          Copyright © 2022 All rights reserved by Sourcier Ltd.
        </p>
      </div>
      <div
        className="md:ml-auto md:text-right"
      >
        <p>
          Projects on
           
          <a
            className="underline link-accent"
            href="https://github.com/sourcier"
            rel="noopener noreferrer"
            target="_blank"
          >
            Github
          </a>
        </p>
        <p>
          Follow on
           
          <a
            className="underline link-accent"
            href="https://twitter.com/sourcier"
            rel="noopener noreferrer"
            target="_blank"
          >
            Twitter
          </a>
        </p>
        <p>
          Contact via
           
          <a
            className="underline link-accent"
            href="mailto:hello@sourcier.uk"
          >
            Email
          </a>
        </p>
        <p>
          <a
            className="underline link-accent"
            href="/cookie-policy"
          >
            Cookie Policy
          </a>
        </p>
        <p>
          <a
            className="underline link-accent"
            href="/cookie-policy"
          >
            Privacy Policy
          </a>
        </p>
        <p>
          <a
            className="underline link-accent"
            href="/terms-and-conditions"
          >
            Terms & Conditions
          </a>
        </p>
      </div>
    </div>
  </div>
</div>
`;
